tutorials/katacoda/thanos/4-receiver-agent/step2.md

```diff
@@ -0,0 +1,100 @@
+
+# Setup `batmobile` and `batcopter` lightweight metric collection
+
+With `Wayne Enterprise` platform ready to ingest Remote Write data, we need to think about how we satisfy our three requirements:
+
+1. Implement a global view of this data. `Wayne Enterprises` needs to know what is happening in all company parts - including secret ones!
+2. `Batmobile` and `Batcopter` can be out of network for some duration of time. You don't want to lose precious data.
+3. `Batmobile` and `Batcopter` do not have large compute power, so you want an efficient solution that avoids extra computations and storage.
+
+How are we going to do this?
+
+Previously our recommended option was to deploy Prometheus on our edge place that scrapes required applications, and remote writes all metrics to Thanos Receive. This will work great and satisfy 1st and 2nd requirements, but there are a couple of things that Prometheus offers that we don't need in this particular scenario:
+
+* We don't want to alert locally or use recording rules from our edge places.
+* We don't want to query data locally, and we want to store it for an as short duration as possible.
+
+Prometheus was designed as a stateful time-series database, and it adds certain mechanics which are not desired for full forward mode. For example:
+
+* Prometheus builds additional memory structures for easy querying from memory.
+* Prometheus does not remove data when it is safely sent via remote write. It waits for at least two hours, and only after the TSDB block is persistent in the disk, it may or may not remove it depending on retention configuration.
```

Comment:
```suggestion
* Prometheus does not remove data when it is safely sent via remote write. It waits for at least two hours and only after the TSDB block is persisted to the disk, it may or may not be removed, depending on retention configuration.
```